src/util/index.ts

```diff
@@ -9,7 +9,7 @@ export const dealRandomAssignees = (assignees: string, randomTo: string | void):
   return arr;
 };
 
-export const matchKeyword = (content: string, keywords: string[]): boolean => {
+export const matchKeyword = (content: string = '', keywords: string[]): boolean => {
```

Comment:
👀

Reply:
？？？

Reply:
This actually didn't resolve anything. The issue still exists. 
`Error: [🚨 AC] Cannot read properties of null (reading 'toLowerCase')` on v3.3.2 for issue with empty body and 
```sh
...
        with:
          actions: 'check-issue'
          body-includes: '###,Overview'
```